test/skimNtuple2018.cpp

```diff
@@ -90,8 +99,8 @@ const float DYscale_MTT_vHighPt[3] = {0.820, 1.999  , 0.897  };
 ** = then sigma_i = f_i * sigmal_LO
 ** - stitchWeight (0jet) = f_0 / (f_0 * N_inclusive)
 ** - stitchWeight (njet) = f_n / (f_n * N_inclusive + N_njets)
-*/ 
-// computed for Legacy2016, 23 Jan 2020
+*/
+// Legacy2016 (23 Jan 2020) - These are not used in the skimming since the DY sample is already split in jet bins and the weight is included in the XS
```

Comment:
@jonamotta they are used in the skimming both for 2016 and 2018, but not for 2017. Can you remove the comment? "These are non used..." (however they need to be updated for each production, so we'll update these numbers before the next round of skims)